tests/chart_tests/test_nginx_service.py

```diff
@@ -235,3 +235,58 @@ def test_nginx_backend_serviceaccount_defaults(self):
         )[0]
 
         assert "release-name-nginx-default-backend" == doc["metadata"]["name"]
+
+    def test_nginx_min_ready_seconds_defaults(self):
+        doc = render_chart(
+            values={},
+            show_only=["charts/nginx/templates/nginx-deployment.yaml"],
+        )[0]
+
+        assert doc["spec"]["minReadySeconds"] == 0
+
+    def test_nginx_min_ready_seconds_overrides(self):
+        minReadySeconds = 300
+        doc = render_chart(
+            values={"nginx": {"minReadySeconds": minReadySeconds}},
+            show_only=["charts/nginx/templates/nginx-deployment.yaml"],
+        )[0]
+
+        assert doc["spec"]["minReadySeconds"] == minReadySeconds
+
+    def test_nginx_election_ttl_defaults(self):
```

Comment:
We can combine these _defaults test cases into a single test case that tests all defaults.

Reply:
i was adding these at different timelines i can combine those into one